packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts

```diff
@@ -1883,13 +1904,13 @@ const useEchartsCartesianConfig = (
                             itemsMap,
                         );
 
-                        return `${tooltipHeader}<br/><table>${tooltipRows}</table>`;
+                        return `${tooltipHeader}<br/>${tooltipHtml}<table>${tooltipRows}</table>`;
```

Comment:
we add the tooltip to the header. 

Could this generate a vulnerability issue ?

Reply:
Fixed by sanitizing input using dompurify 

![Screenshot from 2025-04-07 13-20-38](https://github.com/user-attachments/assets/a6c53998-a159-4c55-93ad-963e5b7d7e46)